doc/Administration-guide/System-installation/Building-Kaa-server-from-source-code/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: page
+title: Building Kaa server from source code
+permalink: /:path/
+sort_idx: 10
+---
+
+* TOC
+{:toc}
+
+{% include variables.md %}
+
+This section describes how to build the [Kaa server]({{root_url}}Glossary/#) from the source code [available on GitHub]({{github_url}}).
+
+Before building the Kaa server from source, make sure that [Oracle JDK 8](http://www.oracle.com/) and [Apache Maven](https://maven.apache.org/) are installed on your machine.
+
+## Fetching source code
+
+You can use any Git client to fetch Kaa source code from the repository.
+
+To download Kaa repository, run the command below.
+
+```
+$ git clone https://github.com/kaaproject/kaa.git
+```
+
+To build Kaa node Debian/RPM packages, run the following command.
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#debian1">Debian</a></li>
+  <li><a data-toggle="tab" href="#rpm1">RPM</a></li>
+</ul>
+
+<div class="tab-content"><div id="debian1" class="tab-pane fade in active" markdown="1" >
+
+```
+$ mvn -P compile-gwt,mongo-dao,mariadb-dao clean install verify
+```
+
+</div><div id="rpm1" class="tab-pane fade" markdown="1" >
+
+```
+$ mvn -P compile-gwt,mongo-dao,mariadb-dao,build-rpm clean install verify
+```
+
+</div>
+</div>
+
+The Debian build will work correctly on both Linux and Windows operation systems, while the RPM build will only work on Linux with the RPM tool installed.
```

Comment:
I'm not sure what author meant here. I deleted the rpm part since it's up to users which file manager they should handle their files with, so no need for excessive information in this sentence.